packages/frontend/src/components/dialogs/EditAccountAndPasswordDialog.tsx

```diff
@@ -83,37 +85,45 @@ function EditAccountInner(onClose: DialogProps['onClose']) {
   const onUpdate = useCallback(async () => {
     const onSuccess = () => onClose()
 
+    const proxyUpdated =
+      initialSettings.proxyEnabled !== accountSettings.proxyEnabled ||
+      initialSettings.proxyUrl !== accountSettings.proxyUrl
+
     const update = () => {
       openDialog(ConfigureProgressDialog, {
         credentials: accountSettings,
         onSuccess,
         onFail: error => {
           openDialog(AlertDialog, { message: error })
         },
+        proxyUpdated,
       })
     }
 
-    if (initial_settings.addr !== accountSettings.addr) {
+    if (initialSettings.addr !== accountSettings.addr) {
       const confirmed = await openConfirmationDialog({
         confirmLabel: tx('perm_continue'),
         isConfirmDanger: true,
         message: tx('aeap_explanation', [
```

Comment:
The issue says

> Remove the dialog about AEAP (which shows the aeap_explanation translated string), because AEAP will be removed. I did it for DC Android here: https://github.com/deltachat/deltachat-android/pull/3698

Reply:
I already mentioned that in DC dev chat: we will remove the feature in the UI but that is another PR.